thread/src/main/kotlin/io/github/paulgriffith/kindling/thread/FilterList.kt

```diff
@@ -47,6 +63,20 @@ class FilterList(private val emptyLabel: String) : CheckBoxList(FilterModel(empt
                 else -> value.toString()
             }
         }
+
+        object : ListSearchable(this) {
+            init {
+                isCaseSensitive = false
+                isRepeats = true
+                isCountMatch = true
+            }
+
+            override fun convertElementToString(element: Any?): String = element.toString()
+
+            override fun setSelectedIndex(index: Int, incremental: Boolean) {
+                checkBoxListSelectedIndex = index
+            }
+        }
```

Comment:
This allows you to, while the list is in focus, type in and quickly 'filter' down to a specific item, including wildcards (? and *) - a nice to have along the lines of the sorting, I figured